launch_ros/launch_ros/actions/lifecycle_node.py

```diff
@@ -42,8 +42,9 @@ class LifecycleNode(Node):
     def __init__(
         self,
         *,
-        name: Optional[Text] = None,
-        node_name: Optional[Text] = None,
+        name: Optional[SomeSubstitutionsType] = None,
+        namespace: SomeSubstitutionsType = '',
```

Comment:
this should be the same:
```suggestion
        namespace: SomeSubstitutionsType = None,
```

Reply:
7ddcc1a